testData/ide/formatter/tokenFormatted.rho

```diff
@@ -0,0 +1,150 @@
+/**
+* A token contract that implements the ERC20 standard.
+* This contract demonstrates the creation of a token,
+* the transferring of tokens, the querying of a balance,
+* and the printing of the balance query result.
+*/
+
+new token in {
+    contract token (message_sender, total_supply, return_channel) = {
+        new allowed_channel, balances_channel, owner_channel in {
+            allowed_channel!(Map()) |
+            owner_channel!(message_sender) |
+            balances_channel!(Map()) |
+            for (balances <- balances_channel) {
+                balances_channel!( balances.insert(message_sender, total_supply) )
+            } |
+            new transfer, transfer_from, approve, balance_of, allowance, contract_owner in {
+                contract transfer(message_sender, to, amount, rtn) = {
+                    for (balances <- balances_channel) {
+                        new x, y in {
+                            x!(balances.getOrElse(message_sender, 0)) |
+                            y!(balances.getOrElse(to, 0)) |
+                            for ( balance_of_sender <- x; balance_of_receiver <- y; balacen_of_test<-z) {
+                                // Preconditions to check before the transfer goes through
+                                match [
+                                  balance_of_sender >= amount,
+                                  amount > 0,
+                                  balance_of_receiver + amount > balance_of_receiver
+                                ] with
```

Comment:
Auto format will always put `] with` in a new line. It doesn't looks nice if `match [...] with` is in one line. Is this easily fixable?

Reply:
Yes, it can be fixed.